pkg/controller/utils/clientwrapper.go

```diff
@@ -0,0 +1,82 @@
+package utils
```

Comment:
This file largely cribbed from [hive](https://github.com/openshift/hive/blob/68fa75297dbfe1a3911bb3e880a9ae0a746d8c75/pkg/controller/utils/clientwrapper.go).